code/modules/mob/living/carbon/alien/humanoid/update_icons.dm

```diff
@@ -53,7 +53,7 @@
 
 /mob/living/carbon/alien/humanoid/update_transform() //The old method of updating lying/standing was update_icons(). Aliens still expect that.
 	if(lying > 0)
-		lying = 90 //Anything else looks retarded
+		lying = 90 //Anything else looks [censored for github]
```

Comment:
Just use a friendlier counterpart of the vocable, this choice of words doesn't do justice to the great git hosting platform it actually is.

Reply:
Just put in "silly".